api/src/db/migrations/v144_add_raffle_bingo_games.ts

```diff
@@ -805,3 +805,20 @@ export const generatev144TaxFilingData = (overrides: Partial<v144TaxFilingData>)
     ...overrides,
   };
 };
+
+export const generatev144LicenseDetails = (overrides: Partial<v144LicenseDetails>): v144LicenseDetails => {
+  return {
+    nameAndAddress: {
```

Comment:
Pebble - all the generators in this file generate a random string, what's the though on using empty strings? license statues should also be a random value / or randomly selected from the list of possible statuses